app/react/components/Icon.tsx

```
@@ -57,27 +69,25 @@ export function Icon({ icon, feather, className, mode, size }: Props) {
       <Svg
         icon={svgIcon as keyof typeof SvgIcons}
         className={clsx(classes, '!flex')}
+        aria-label={ariaLabel}
       />
     );
   }
 
-  if (feather) {
-    const iconName = icon
-      .split('-')
-      .map((s) => s.slice(0, 1).toUpperCase() + s.slice(1))
-      .join('') as keyof typeof featherIcons;
-    const IconComponent = featherIcons[iconName];
-    if (!IconComponent) {
-      throw new Error(`Feather icon not found: ${iconName}`);
-    }
-    return <IconComponent className={classes} />;
+  const iconName = icon
+    .split('-')
+    .map((s) => s.slice(0, 1).toUpperCase() + s.slice(1))
+    .join('') as keyof typeof featherIcons;
+  const IconComponent = featherIcons[iconName];
+  if (!IconComponent) {
+    return (
+      <Svg
+        icon={'placeholder' as keyof typeof SvgIcons}
+        className={clsx(classes, inline ? '!inline' : '!flex')}
```

inline is only for the placeholder?

---

I actually think you should throw an error here. this way the developer would notice using a wrong icon. when do we expect to need to use the placeholder?

---

Inline as a prop is passed over from EE - I updated the SVG component to match EE too
```
  if (icon.indexOf('svg-') === 0) {
    const svgIcon = icon.replace('svg-', '');
    return (
      <Svg
        icon={svgIcon as keyof typeof SvgIcons}
        className={clsx(classes, inline ? '!inline' : '!flex')}
        aria-label={ariaLabel}
      />
    );
  }
```